services/workflows-service/scripts/workflows/full_kyb_process.ts

```diff
@@ -0,0 +1,310 @@
+import { PrismaClient } from '@prisma/client';
+import { defaultContextSchema, StateTag } from '@ballerine/common';
+
+import { env } from '../../src/env';
+import { kycEmailSessionDefinition } from './kyc-email-process-example';
+import { parentKybWithSessionWorkflowDefinition } from './parent-kyb-kyc-session-workflow';
+
+export const fullKybProcessWorkflowDefinition = {
+  id: 'full_kyb_process',
+  name: 'full_kyb_process',
+  version: 1,
+  definitionType: 'statechart-json',
+  definition: {
+    id: 'full_kyb_process_v1',
+    predictableActionArguments: true,
+    initial: 'data_collection',
+    context: {
+      documents: [],
+    },
+    states: {
+      data_collection: {
+        tags: [StateTag.COLLECTION_FLOW],
+        on: {
+          start: [
+            {
+              target: 'run_cn_kyb',
+              cond: {
+                type: 'jmespath',
+                options: {
+                  rule: 'entity.data.additionalInfo.country == `cn`', // @TODO: Work out companyNumber with Daniel
+                },
+              },
+            },
+            {
+              target: 'run_hk_kyb',
+              cond: {
+                type: 'jmespath',
+                options: {
+                  rule: 'entity.data.additionalInfo.country == `hk`', // @TODO: Work out companyNumber with Daniel
+                },
+              },
+            },
+            {
+              target: 'auto_reject',
+            },
+          ],
+        },
+      },
+      run_cn_kyb: {
+        tags: [StateTag.PENDING_PROCESS],
+        on: {
+          // CONTINUE: [{ target: 'run_aml' }],
+          CONTINUE: [{ target: 'run_vendor_data_collection' }],
+          FAILED: [{ target: 'auto_reject' }],
+        },
+      },
+      run_hk_kyb: {
+        tags: [StateTag.PENDING_PROCESS],
+        on: {
+          // CONTINUE: [{ target: 'run_aml' }],
+          CONTINUE: [{ target: 'run_vendor_data_collection' }],
+          FAILED: [{ target: 'auto_reject' }],
```

Comment:
It shouldn't go to auto_reject, it should move to "failure" state from there we should implement recovery actions